transcom/mymove
# B 21904 nts oconus int

## [B-21904](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1038106)

## Summary

This PR adds in the pre approved service items for international NTS shipments.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as a TOO
2. Add an international NTS shipment to a move. Test the variations of the move:
OCONUS -> OCONUS
OCONUS -> CONUS... etc
3. Click on `Approve and send`.
4. Confirm on the next page`International Shipping and Linehaul` and `International Packing with NTS Pricing Factor` are preapproved service items. `Fuel Surcharge - PODFSC` should only be pre approved if the shipment is OCONUS -> CONUS

Note: OCONUS -> CONUS etc are determined by the shipments pickupaddress and storage facility address.
Example: NTS Shipment
**Pickup Address**     **->  _Storage Facility Address_**         ==      **Ordering Type** 
              Alaska              ->                          Alaska               ==                   OCONUS -> OCONUS
              Iowa                 ->           Alaska    ==               CONUS -> OCONUS
### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
_**OCONUS -> CONUS**_
<img width="1020" alt="Screenshot 2024-12-12 at 4 02 50 PM" src="https://github.com/user-attachments/assets/bbd5bb66-5f18-40e5-806d-88cf6ae9a471" />

_**OCONUS -> OCONUS**_
<img width="1039" alt="Screenshot 2024-12-12 at 4 01 45 PM" src="https://github.com/user-attachments/assets/48893c39-2bc1-47f4-88ef-f5c1c8c258d6" />
